packages/react-admin/src/reducer/admin/references/possibleValues.js

```diff
@@ -16,22 +16,16 @@ export default (previousState = initialState, { type, payload, meta }) => {
 
 export const getPossibleReferences = (
     state,
-    referenceSource,
-    reference,
+    possibleValues,
```

Comment:
I don't understand this change. Could you explain why you do it? Could you add a unit test?

Reply:
`getPossibleReferences` no longer receives the entire state, but a selection of the state retrieved by a memoized selector in `ReferenceInput`. This safes unnecessary rerenders.

Reply:
then rename the `state` param to something more significant.